maintainers.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: simple-page
+title: RIOT Maintainers
+subtitle: RIOT is nurtured by a strong community of maintainers.
```

Comment:
RIOT maintainers care about RIOT helping RIOT growing and improving.

Reply:
"RIOT RIOT RIOT" ;-)? How about „Our maintainers care about RIOT and help the operating system to grow and improve."?

Reply:
fine with me, but i would replace "operating system" by "RIOT". or we just shorten "Our maintainers help RIOT to grow and improve."

Reply:
@miri64 but this change is not included?

Reply:
Sorry, yes. Just noticed that myself.

Reply:
> "Our maintainers help RIOT to grow and improve."

Took that one.